### ISSUE TITLE
Feature Request: Force front-end password reset path for CP users

### ISSUE THREAD

**Issue by jakepm** [craftcms/cms#6056]
New config setting, to modify the 'reset password' behaviour.

I can see that if a user has access to the control panel, that their 'reset password' link will direct to the control panel reset page, rather than whatever was set via the setPasswordPath config setting.

I understand the logic here, but could you add another config setting that forces all password resets to the setPasswordPath template, rather than treating users with CP access differently.

eg. setPasswordPathForced => true

**Comment by brandonkelly** [comment]
What’s the use case?

**Comment by jakepm** [comment]
Just to white-label the entire authentication experience for the site owner. They are aware that the site is running on Craft and do use the control panel, but my preference would be for them to see the same login and authentication workflows (forgot/reset) that their users see. 

This is easy to achieve for the login page and forgot password pages, as I have provided the front-end white-labelled URLs to them. However, if they reset their password, they are taken to the Craft 'set password' page, rather than the white-labelled version that I have created for non-CP users.
